### PR TITLE
fix integration test document

### LINKED ISSUE #23946: Failed to run Docker mode according to the integration test document
## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?
master

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy

### Expected behavior
image

Execute commands correctly

### Actual behavior
image

### Reason analyze (If you can)
pom.xml path error in document .The correct command is
```
./mvnw -B clean install -f test/e2e/pom.xml -Pit.env.docker -Dit.cluster.adapters=proxy,jdbc -Dit.scenarios=${scenario_name_1,scenario_name_2,scenario_name_n} -Dit.cluster.databases=MySQL
```
### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
Run integration test with docker mode

### PR DESCRIPTION
**Pull request by lushaorong**
Signed-off-by: lushaorong <wwwwwlsr@163.com>

Fixes #23946.

Changes proposed in this pull request:
  - fix integration test document

---

Before committing this PR, I'm sure that I have checked the following options:
- [ ] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [ ] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [ ] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: docs/document/content/test-manual/integration-test/_index.cn.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ weight = 1
 
 ## 使用指南
 
-模块路径：`shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-test-suite`
+模块路径：`shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite`
```
Comment (sandynz) [inline docs/document/content/test-manual/integration-test/_index.cn.md:52]: Looks the module path doesn't exist. Is it `test/e2e/suite`?

File: docs/document/content/test-manual/integration-test/_index.cn.md [hunk 2]
```diff
@@ -156,14 +156,14 @@ it.cluster.databases=H2,MySQL,Oracle,SQLServer,PostgreSQL
 #### 运行 Docker 模式
 
 ```bash
-./mvnw -B clean install -f shardingsphere-test/shardingsphere-test-e2e/pom.xml -Pit.env.docker -Dit.cluster.adapters=proxy,jdbc -Dit.scenarios=${scenario_name_1,scenario_name_2,scenario_name_n} -Dit.cluster.databases=MySQL
+./mvnw -B clean install -f test/e2e/pom.xml -Pit.env.docker -Dit.cluster.adapters=proxy,jdbc -Dit.scenarios=${scenario_name_1,scenario_name_2,scenario_name_n} -Dit.cluster.databases=MySQL
```
Comment (sandynz) [inline docs/document/content/test-manual/integration-test/_index.cn.md:159]: 1, Is `-f test/e2e/pom.xml` or `-f test/e2e/suite/pom.xml`?

2, Looks this command is not used to build docker image. Could we refer to `e2e.yml` and verify it?

**Review by sandynz — CHANGES_REQUESTED** [review]

**Comment by taojintianxia** [comment]
please refactor the document as @sandynz 's suggestion.
some paths metioned in this PR have been changed

File: docs/document/content/test-manual/integration-test/_index.cn.md (hunk 1 above)
Comment (lushaorong) [reply docs/document/content/test-manual/integration-test/_index.cn.md:52]: > Looks the module path doesn't exist. Is it `test/e2e/suite`?

I think this is ```module path```, so I wrote ```shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite```

Comment (sandynz) [reply docs/document/content/test-manual/integration-test/_index.cn.md:52]: Currently, module related local directory name is different with artifactId.

I tested on Intellij IDEA "Navigate by File":
1. Navigate by `shardingsphere-test/shardingsphere-test-e2e/shardingsphere-test-e2e-suite`: No any file shown.
2. Navigate by `test/e2e/suite`: It shows `shardingsphere-test-e2e-suite` module and related files.

And also on local file system, users could locate the module by `test/e2e/suite`.

So I think use `test/e2e/suite` could benefit for users.
If `module path` is confusing, then we could consider to change these words.

**Pull request closed without merging**